src/Umbraco.Core/Persistence/Repositories/DataTypeDefinitionRepository.cs

```diff
@@ -276,47 +276,28 @@ protected override void PersistDeletedItem(IDataTypeDefinition entity)
 
         public PreValueCollection GetPreValuesCollectionByDataTypeId(int dataTypeId)
         {
-            var cached = RuntimeCache.GetCacheItemsByKeySearch<PreValueCollection>(GetPrefixedCacheKey(dataTypeId));
-            if (cached != null && cached.Any())
-            {
-                //return from the cache, ensure it's a cloned result
-                return (PreValueCollection)cached.First().DeepClone();
-            }
-
-            return GetAndCachePreValueCollection(dataTypeId);
-        }
-
-        internal static string GetCacheKeyRegex(int preValueId)
-        {
-            return CacheKeys.DataTypePreValuesCacheKey + @"[-\d]+-([\d]*,)*" + preValueId + @"(?!\d)[,\d$]*";
+            var collection = GetCachedPreValueCollection(dataTypeId);
+            return collection == null ? null : (PreValueCollection) collection.DeepClone();

```

Comment:
Turns out we've been double Deep Cloning here, we don't need to call DeepClone because the RuntimeCache is a Deep Clone cache provider and will already deep clone the PreValueCollection out of it, this doesn't need to be here.